src/test/scala-2.11/markets/participants/strategies/TradingStrategySpec.scala

```diff
@@ -0,0 +1,46 @@
+package markets.participants.strategies
+
+import akka.agent.Agent
+
+import markets.tickers.Tick
+import markets.tradables.{Security, Tradable}
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.collection.mutable
+import scala.concurrent.ExecutionContext.Implicits.global
+
+
+class TradingStrategySpec extends FlatSpec with Matchers {
+
+  val tradable = Security("GOOG")
+  val tickers = mutable.Map[Tradable, Agent[Tick]](tradable -> Agent(Tick(1, 1, 1, 1, 1)))
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2363386122/issues/source?bid=2921361&fileBranchId=3219434#l16)